seyik/app.js

```diff
@@ -111,6 +110,57 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/modify", async (req, res) => {
+  const { postingContent, user_id } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+    content = ?
+    WHERE posts.user_id = ? 
+    `,
+    [postingContent, user_id]
+  );
+  const rows = await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.name as userName,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+      FROM users LEFT JOIN posts
+      ON posts.user_id = users.id
+  `
+  );
+  res.status(200).json({ data: rows });
```

Comment:
```suggestion
  return res.status(201).json({ data: rows });
```
PATCH라는 method와 API 내부 로직을 생각해본다면 단순 응답을 나타내는 status code인 200보다 적절한 상태코드를 사용할 수 있을 것 같습니다.